test/libdnf/base/test_goal.cpp

```diff
@@ -35,7 +35,7 @@ void BaseGoalTest::setUp() {
 }
 
 void BaseGoalTest::test_install() {
-    libdnf::Goal goal(base.get());
+    libdnf::Goal goal(*base.get());
```

Comment:
Please use `*base` instead of `*base.get()`. Repeats in multiple places in the file.